projects/storefrontapp-e2e-cypress/cypress/integration/regression/customer-ticketing/create-ticket.e2e-spec.ts

```diff
@@ -140,6 +147,51 @@ describe('ticketing', () => {
         customerTicketing.verifyTicketDoesNotExist(testTicketDetails);
       });
 
+      it('should not create ticket if subject exceeds 255 character limit', () => {
+        const TICKET_SUBJECT_MAX_LENGTH = 255;
+        const testTicketDetails: TestTicketDetails = {
+          subject: new Array(TICKET_SUBJECT_MAX_LENGTH + 1).join('a'),
+          message: 'Exceeding character limit',
+          category: TestCategory.complaint,
+        };
+        customerTicketing.loginRegisteredUser();
+        customerTicketing.visitElectronicTicketListingPage();
+        customerTicketing.openCreateTicketPopup();
+        customerTicketing.fillTicketDetails(testTicketDetails);
+        customerTicketing.clickSubmit();
+        customerTicketing.verifyTicketSubjectAndMessageDoNotExceedCharacterLimit();
+      });
+
+      it('should not create ticket if message exceeds 5000 character limit', () => {
+        const TICKET_MESSAGE_MAX_LENGTH = 5000;
+        const testTicketDetails: TestTicketDetails = {
+          subject: 'Exceeding character limit',
+          message: new Array(TICKET_MESSAGE_MAX_LENGTH + 1).join('a'),
```

Comment:
nit: you could extract this line into a function:

export function generateDummyStringOfLength(length: number): string {
  return new Array(length).join('a')
}

usage:
...
message: generateDummyStringOfLength(TICKET_MESSAGE_MAX_LENGTH + 1),